EDITS: Minor edits made to content in run-access-cm.md (#588)
ACCESS-Hive/access-hive.github.io: docs/models/run-a-model/run-access-cm.md

<!-- End of tab content -->

<div class="note">

>     The workflow to run ACCESS-CM is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.

Reviewer: ```suggestion
    The workflow to run {{ model }} is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
```

    <br>
    The above tabs allow you choose the type of workflow you would prefer to follow.
    <br>
    <br>

>     If you are new to ACCESS-CM, we <b>recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.

Reviewer: ```suggestion
    If you are new to ACCESS-CM, we <b>strongly recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.
```

>     If you are new to ACCESS-CM, we <b>recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.

Reviewer: ```suggestion
    If you are new to {{ model }}, we <b>recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.
```

    <!-- ARE/Gadi-->
    <div>
        <div class="note">

>             Your chosen workflow is ARE/<i>Gadi</i>. If you would prefer to run {{ model }} on <i>accessdev</i>, then select the <a href="#accessdev"><i>accessdev</i> workflow</a>.

Reviewer: ```suggestion
            Your chosen workflow is ARE / <i>Gadi</i>. If you would prefer to run {{ model }} on <i>accessdev</i>, then select the <a href="#accessdev"><i>accessdev</i> workflow</a>.
```

I think leaving space makes it more readabale. Similarly to what is done in other similar parts of the page.

                This means that the ARE VDI session only needs to carry out setup steps as well as starting the run itself. All these tasks can be easily done with only 1 CPU.
            </li>
            <li>
                <b>Project</b> &rarr; a project of which you are a member.
                <br>
                The project must have allocated <i>Service Units</i> (SU) to run your simulation. Usually, but not always, this corresponds to your <code>$PROJECT</code>.
                <br>
                For more information, refer to how to <a href="/getting_started/first_steps#join-relevant-nci-projects">Join relevant NCI projects</a>.
            </li>
            <li>
                <b>Storage</b> &rarr; <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32</code> (minimum)
                <br>
                This is a list of all project data storage (joined by <code>+</code> signs) needed for the {{ model }} simulation. In ARE, storage locations need to be explicitly defined to access data from within a VDI instance.
                <br>

>                 An {{ model }} simulation can be unique and input data can originate from various sources. Hence, if your simulation requires data originating from projects other than <code>access</code>, <code>hh5</code>, <code>hr22</code> or <code>ki32</code>, you need to add those projects to the storage path.

Reviewer: ```suggestion
                Every {{ model }} simulation can be unique and input data can originate from various sources. Hence, if your simulation requires data stored in project folders other than <code>access</code>, <code>hh5</code>, <code>hr22</code> or <code>ki32</code>, you need to add those projects to the storage path.
```

                <br>

>                 For example, if your {{ model }} simulation requires data from <code>/g/data/tm70</code> and <code>/scratch/w40</code>, your full storage path will be: <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32<b>+/gdata/tm70+scratch/w40</b></code>

Reviewer: ```suggestion
                For example, if your {{ model }} simulation requires data stored in <code>/g/data/tm70</code> and <code>/scratch/w40</code>, your full storage path will be: <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32<b>+/gdata/tm70+scratch/w40</b></code>
```

            </li>
        </ul>

>         Now, launch an ARE session and, once it starts, click on <i>Launch VDI Desktop</i>.

Reviewer: ```suggestion
        Launch the ARE session and, once it starts, click on <i>Launch VDI Desktop</i>.
```

        <img src="/assets/run_access_cm/launch_are_vdi.gif" alt="Launch ARE VDI session" class="example-img" loading="lazy"/>
        <br>
        <h3>Open the terminal</h3>
        Once the new tab opens, you will see a Desktop with a few folders on the left.
        <br>
        To open the terminal, click on the black terminal icon at the top of the window. You should now be connected to a <i>Gadi</i> computing node.
        <img src="/assets/run_access_cm/open_are_vdi_terminal.gif" alt="Open ARE VDI terminal" class="example-img" loading="lazy"/>
    </div>
    <!-- accessdev-->
    <div>
        <div class="note">
            Your chosen workflow is <i>accessdev</i>. If you would prefer to run {{ model }} on ARE / <i>Gadi</i>, then select the <a href="#are">ARE / <i>Gadi</i> workflow</a>.
        </div>
        Login to <i>accessdev</i> by runnning:
        <pre><code>ssh accessdev</code></pre>
        <div class="note">

>             If you have not yet setup your <i>accessdev</i> connection through `ssh`, please check the <a href="https://accessdev.nci.org.au/trac/wiki/GettingConnected">Getting Connected to Accessdev</i></a> guide.

Reviewer: ```suggestion
            If you have not yet set up your <i>accessdev</i> connection through `ssh`, please check the <a href="https://accessdev.nci.org.au/trac/wiki/GettingConnected">Getting Connected to Accessdev</i></a> guide.
```

I think the verb is **set up**, not to be confused with the noun/adjective **setup** (ref. --> https://www.future-perfect.co.uk/grammar-tips/easy-mistakes/is-it-setup-set-up-or-set-up).

Author: ARE/Gadi was suggested for consistentcy with other uses of this term on the page. But, I am fine with your suggestion.

                <br>
                <ul>
                    <li>

>                         <i>Local copy</i>

Reviewer: ```suggestion
                        <i>Local-only copy</i>
```
I think it needs to be stressed that is only a local copy.
